packages/yourloops/lib/custom-hooks.ts

```diff
@@ -0,0 +1,7 @@
+import { useMemo } from 'react'
```

Comment:
I propose to create a folder called it custom-hooks and create file called useQueryParams